packages/core/test/SignUp.ts

```diff
@@ -180,17 +180,36 @@ describe('Signup', function () {
             const newUsername4 = genRandomSalt().valueOf()
             const accounts = await ethers.getSigners()
 
-            const tx = await unirepSocialContract
+            const tx1 = await unirepSocialContract
```

Comment:
This is just a note, instead of using like `tx1/tx2` variables you can put this logic inside a block like this:

```js
...
const newUsername4 = genRandomSalt().valueOf()
const accounts = await ethers.getSigners()
{
  const tx = await unirepSocialContract....
  const receipt = await tx.wait()
  expect(receipt.status)...
}
{
  const tx = await unirepSocialContract....
  const receipt = await tx.wait()
  expect(receipt.status)...
}
```

This bracket blocks can exist anywhere in the code, not just around `if` or `for` or functions or other flow control structures. Then the variables are scoped to that block and gone after.